src/stcal/resample/utils.py

```diff
@@ -74,15 +74,16 @@ def build_driz_weight(model, weight_type=None, good_bits=None,
     model : dict
         Input model: a dictionary of relevant keywords and values.
 
-    weight_type : {"exptime", "ivm"}, None, optional
+    weight_type : {"exptime", "ivm", "ivm-sky"}, None, optional
         The weighting type for adding models' data. For
         ``weight_type="ivm"``, the weighting will be
         determined per-pixel using the inverse of the read noise
-        (VAR_RNOISE) array stored in each input image. If the
-        ``VAR_RNOISE`` array does not exist,
-        the variance is set to 1 for all pixels (i.e., equal weighting).
-        If ``weight_type="exptime"``, the weight will be set equal
-        to the measurement time when available and to
+        (VAR_RNOISE) array stored in each input image. If
+        ``weight_type="ivm-sky"``, the weighting will be determined
+        by the inverse of the sky variance ``VAR_SKY``. If the ``VAR_SKY``
+        array does not exist, the variance is set to 0 for all pixels
+        (i.e., equal weighting).  If ``weight_type="exptime"``, the 
+        weight will be set equal to the measurement time when available and to
```

Comment:
For this and the corresponding documentation in resample.py let's merge the ivm and ivm-sky documentation.
i.e.,
The weighting type for adding models' data.  For weight_type = ivm and weight_type = ivm-sky, the weighting will be determined per pixel using the inverse of either the read noise or sky noise arrays, respectively (VAR_RNOISE or VAR_SKY).  If the array does not exist, the weight is set to 1 for all pixels (i.e., equal weighting).

Reply:
Done.